python/cudf/cudf/core/frame.py

```diff
@@ -158,6 +158,70 @@ def _drop_na_rows(self, how="any", subset=None, thresh=None):
         result._copy_categories(self)
         return result
 
+    def interleave_columns(self):
+        """
+       Interleave Series columns of a table into a single column.
+
+        Converts the column major table `cols` into a row major column.
+        Parameters
+        ----------
+        cols : input Table containing columns to interleave.
+
+        Example
+        -------
+        >>> df = DataFrame([['A1', 'A2', 'A3'], ['B1', 'B2', 'B3']])
+        >>> df
+        0    [A1, A2, A3]
+        1    [B1, B2, B3]
+        >>> df.interleave_columns()
+        0    A1
+        1    B1
+        2    A2
+        3    B2
+        4    A3
+        5    B3
+
+        Returns
+        -------
+        The interleaved columns as a single column
+        """
+
+        return self._constructor_sliced(
```

Comment:
Use the classmethod to construct from a Table
```suggestion
        return self._constructor_sliced._from_table(
```

Reply:
`interleave_columns` returns a column, so I think this is correct as-is

Reply:
Woops!